movies.py

```diff
@@ -15,6 +20,7 @@ def add_movie():
         'year': get_year
         }
     )
+    logging.info(f"User added a movie {get_title.title()} in {get_year}")
 
 def display_movie():
     for movie in movies:
```

Comment:
You can also remove for loop here if we are just displaying movies. 
```suggestion
    print (dict(zip(movies['title'], movies['year'])))
```

Reply:
And don't forget to add logging :)